[2025-02-12T09:31:39.506+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-12T09:31:39.598+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_payments manual__2025-02-12T09:30:31.798632+00:00 [queued]>
[2025-02-12T09:31:39.625+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_payments manual__2025-02-12T09:30:31.798632+00:00 [queued]>
[2025-02-12T09:31:39.626+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-12T09:31:39.669+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform.transform_dim_payments> on 2025-02-12 09:30:31.798632+00:00
[2025-02-12T09:31:39.681+0000] {standard_task_runner.py:63} INFO - Started process 247 to run task
[2025-02-12T09:31:39.694+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl', 'transform.transform_dim_payments', 'manual__2025-02-12T09:30:31.798632+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/e_commerce_dw_dag.py', '--cfg-path', '/tmp/tmpsrs6k2ro']
[2025-02-12T09:31:39.703+0000] {standard_task_runner.py:91} INFO - Job 234: Subtask transform.transform_dim_payments
[2025-02-12T09:31:39.900+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl.transform.transform_dim_payments manual__2025-02-12T09:30:31.798632+00:00 [running]> on host 18b9093ce731
[2025-02-12T09:31:40.604+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl' AIRFLOW_CTX_TASK_ID='transform.transform_dim_payments' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T09:30:31.798632+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-12T09:30:31.798632+00:00'
[2025-02-12T09:31:40.608+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-12T09:31:40.684+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:31:40.906+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:31:41.161+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:31:41.385+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-02-12T09:31:42.878+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:31:43.347+0000] {logging_mixin.py:188} INFO - Đã transform và lưu dữ liệu vào dim_payments
[2025-02-12T09:31:43.356+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-12T09:31:43.358+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-12T09:31:43.378+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl, task_id=transform.transform_dim_payments, run_id=manual__2025-02-12T09:30:31.798632+00:00, execution_date=20250212T093031, start_date=20250212T093139, end_date=20250212T093143
[2025-02-12T09:31:43.471+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-12T09:31:43.582+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T09:31:43.590+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
